com.github.sakya.corechess.json

```diff
@@ -0,0 +1,111 @@
+        "append-path": "/usr/lib/sdk/dotnet6/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet6/lib",
+        "env": {
+          "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+        },
+        "arch": {
+          "x86_64": {
+            "env": {
+              "RUNTIME": "linux-x64"
+            }
+          }
+        }
+      },
+      "build-commands": [
+        "dotnet publish CoreChess.sln -c Release --source ./nuget-sources --runtime $RUNTIME -p:PublishReadyToRun=true --self-contained --output ./dist/$RUNTIME",
+        "cp CoreChess/Lib/Bass/$RUNTIME/libbass.so ./dist/$RUNTIME/",
+        "cp -r --remove-destination ./dist/$RUNTIME /app/bin/",
+        "mkdir -p /app/share/applications/",
+        "cp ./Flatpak/com.github.sakya.corechess.desktop /app/share/applications/",
+        "mkdir -p /app/share/metainfo/",
+        "cp ./Flatpak/com.github.sakya.corechess.appdata.xml /app/share/metainfo/",
+        "mkdir -p /app/share/icons/hicolor/512x512/apps/",
+        "cp ./icon.png /app/share/icons/hicolor/512x512/apps/com.github.sakya.corechess.png"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "tag": "0.9.4.1",
+          "url": "https://github.com/sakya/corechess.git"
+        },
+        "nuget.json"
+      ]
+    },
+    {
+      "name": "Stockfish",
+      "buildsystem": "simple",
+      "build-options": {
+        "arch": {
+          "x86_64": {
+            "env": {
+              "RUNTIME": "linux-x64"
+            }
+          }
+        }
+      },
+      "build-commands": [
+        "mkdir -p /app/bin/Engines/stockfish",
+        "cp ./stockfish_14.1_linux_x64 /app/bin/Engines/stockfish",
+        "chmod u+x /app/bin/Engines/stockfish/stockfish_14.1_linux_x64",
+        "cp ./nn-13406b1dcbe0.nnue /app/bin/Engines/stockfish"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://stockfishchess.org/files/stockfish_14.1_linux_x64.zip",
+          "sha256": "df5f047f87d628af0c746e5e1e526ef1a463592d41afadbc8cc457012c35ade8"
+        },
+        {
+          "type": "file",
+          "url": "https://tests.stockfishchess.org/api/nn/nn-13406b1dcbe0.nnue",
+          "sha256": "13406b1dcbe0a36dfc5e07da51d8db38b7a45d7da43078ec25a7624c2b83236f"
+        }
+      ]
+    },
+    {
+      "name": "Komodo",
+      "buildsystem": "simple",
+      "build-options": {
+        "arch": {
+          "x86_64": {
+            "env": {
+              "RUNTIME": "linux-x64"
+            }
+          }
+        }
+      },
+      "build-commands": [
+        "mkdir -p /app/bin/Engines/komodo",
+        "cp ./Linux/komodo-12.1.1-linux /app/bin/Engines/komodo",
+        "chmod u+x /app/bin/Engines/komodo/komodo-12.1.1-linux"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://komodochess.com/pub/komodo-12.zip",
```

Comment:
Does the license of Komodo allow redistribution?

Reply:
I'm unsure about this: the zip doesn't include a license, I'm removing komodo.